components/formats-gpl/src/loci/formats/in/FujiReader.java

```diff
@@ -118,7 +118,7 @@ public String[] getSeriesUsedFiles(boolean noPixels) {
     if (noPixels) {
       return new String[] {infFile};
     }
-    return new String[] {infFile, pixelsFile};
+    return new String[] {pixelsFile, infFile};
```

Comment:
While this causes the test to pass, it looks like it shifts the problem to initializing .inf files:

```
$ showinf -no-upgrade -nopix TPN_membraneC_1sec.inf
Checking file format [Fuji LAS 3000]
Initializing reader
FujiReader initializing TPN_membraneC_1sec.inf
Initialization took 0.179s

Reading core metadata
filename = TPN_membraneC_1sec.inf
Used files:
	/home/melissa/data/bf-data-repo/automated-tests/curated/fuji/venu/TPN_membraneC_1sec.img
	/home/melissa/data/bf-data-repo/automated-tests/curated/fuji/venu/TPN_membraneC_1sec.inf
```